spec/datetime.html

```diff
@@ -1,352 +1,824 @@
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateYearMonth(_dateTime_.[[Year]], _dateTime_.[[Month]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.DateTime.prototype.getMonthDay ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _dateTime_ be the *this* value.
+        1. If Type(_dateTime_) is not Object or _dateTime_ does not have an [[InitializedTemporalDateTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateMonthDay(_dateTime_.[[Month]], _dateTime_.[[Day]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.DateTime.prototype.getTime ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _dateTime_ be the *this* value.
+        1. If Type(_dateTime_) is not Object or _dateTime_ does not have an [[InitializedTemporalDateTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Return ? CreateTime(_dateTime_.[[Hour]], _dateTime_.[[Minute]], _dateTime_.[[Second]], _dateTime_.[[Millisecond]], _dateTime_.[[Microsecond]], _dateTime_.[[Nanosecond]]).
+      </emu-alg>
+    </emu-clause>
   </emu-clause>
 
+  <emu-clause id=sec-temporal-datetime-abstract-ops>
+    <h1>Abstract operations</h1>
+
+    <emu-clause id=sec-temporal-ToPartialDateTime aoid=ToPartialDateTime>
+      <h1>ToPartialDateTime ( _dateTime_ )</h1>
+      <emu-alg>
+        1. If Type(_dateTime_) is String, then
+          1. Return ? ParseDateTimeString(_dateTime_).
+        1. If Type(_dateTime_) is Object, then
+          1. Let _result_ be the new Record {
+            [[Year]]: *undefined*,
+            [[Month]]: *undefined*,
+            [[Day]]: *undefined*,
+            [[Hour]]: *undefined*,
+            [[Minute]]: *undefined*,
+            [[Second]]: *undefined*,
+            [[Millisecond]]: *undefined*,
+            [[Microsecond]]: *undefined*,
+            [[Nanosecond]]: *undefined*,
+            }.
+          1. Let _any_ be *false*.
+          1. For each row of <emu-xref href="#table-temporal-datetimelike-properties"></emu-xref>, except the header row, in table order, do
+            1. Let _property_ be the Property value of the current row.
+            1. Let _value_ be ? Get(_dateTime_, _property_).
+            1. If _value_ is not *undefined*, then
+              1. Set _any_ to *true*.
+              1. Set _value_ to ? ToInteger(_value_).
+              1. Set _result_'s internal slot whose name is the Internal Slot value of the current row to _value_.
+          1. If _any_ is *false*, then
+            1. Throw a *TypeError* exception.
+          1. Return _result_.
+        1. Throw a *TypeError* exception.
+      </emu-alg>
+
+      <emu-table id="table-temporal-datetimelike-properties">
+        <emu-caption>Properties of a DateTimeLike</emu-caption>
```

Comment:
This caption doesn’t seem quite accurate

Reply:
Why not?

Reply:
Because the left hand column is internal slots, and because it’s a lookup table of DateTimeLike properties to internal slot names, not just properties of a DateTimeLike.